### Problem 1:
Blood glucose levels for obese patients have a mean of 100 with a standard deviation of15. A researcher thinks that a diet high in raw cornstarch will have a positive effect on
blood glucose levels. A sample of 36 patients who have tried the raw cornstarch diet
have a mean glucose level of 108. Test the hypothesis that the raw cornstarch had an
effect or not.

#### Solution: 
	H0 : μ=100 (null-hypothesis)
    H1 : μ≠100 (alternative-hypothesis)
    
    Population standard deviation is known and the sample is large(≥30), thus, normal distribution is used.
    Assume 95% confidence level

In [1]:
import numpy as np
import scipy.stats as sts
n=36
sigma = 15

Standard_error = sigma / np.sqrt(36)
print("Standard error is : ",Standard_error)

Standard error is :  2.5


In [2]:
x̅ = 108
𝜇 = 100

z_test = (x̅ - 𝜇)/ Standard_error
print("Value of z_test is : ",z_test)

Value of z_test is :  3.2


In [3]:
## We do two tail-test test to check both the sides, significance-level(α)=0.05
print("Value of -0.025 in z-table:",sts.norm.ppf(0.025))
print("Value of +0.025 in z-table:",sts.norm.ppf(1-0.025))

Value of -0.025 in z-table: -1.9599639845400545
Value of +0.025 in z-table: 1.959963984540054


Conclusion: Since the calculated statistic 3.2 is greater than the critical value, 1.96, we reject the null hypothesis and conclude that the raw cornstarch had an effect on blood glucose level.

### Problem 2:
In one state, 52% of the voters are Republicans, and 48% are Democrats. In a second
state, 47% of the voters are Republicans, and 53% are Democrats. Suppose a simple
random sample of 100 voters are surveyed from each state.
What is the probability that the survey will show a greater percentage of Republican
voters in the second state than in the first state?

#### Solution:
Mean Difference in number of  republican  in both states = 0.52  - 0.47  = 0.05

n(random sample) = 100

standard deviation   =   √ ((0.52)(0.48)/100 + (0.47)(0.53)/100 ) => 0.0706

Z score  = ( Value - mean ) / Standard deviation

In [4]:
x̅ = 0
𝜇 = 0.05
n=100
Standard_deviation = np.sqrt((0.52*0.48)/100 + (0.47*0.53)/100)
print("Standard deviation: ",Standard_deviation)

Standard deviation:  0.07061869440877536


In [5]:
z_test = (x̅ - 𝜇)/ Standard_deviation
print("Value of z_test is : ",z_test)

Value of z_test is :  -0.7080278164104207


In [6]:
print("value of -0.708 in z-table is :",sts.norm.cdf(-0.708))

value of -0.708 in z-table is : 0.2394726287398799


Hence 0.24 is the probability that the survey will show a greater percentage of Republican voters in the second  state than in the first state

### Problem 3:
You take the SAT and score 1100. The mean score for the SAT is 1026 and the standard
deviation is 209. How well did you score on the test compared to the average test taker?

#### Solution:
x̅ = 1100

𝜇 = 1026

Standar deviation = 209

Z score = ( Value - mean ) / Standard deviation

In [7]:
x̅ = 1100
𝜇 = 1026
Standard_deviation = 209
z_score = (x̅ - 𝜇) / Standard_deviation
print("Value of z_score is : ",z_score)

Value of z_score is :  0.35406698564593303


In [8]:
print("value of 0.354 in z-table is : ",sts.norm.cdf(0.354))

value of 0.354 in z-table is :  0.6383305583821042


Conclusion: I score 0.6383 or 63.83% in the test as compared to the average test takers.